test.sh

```diff
@@ -26,7 +26,7 @@ fi
 
 echo -n "testing ica.csv... "
 lein run -l $LEDGER -f test/data/ica.csv -F ';' -sa 1 -m 4 -t '%1' \
-	-a 'Assets:ICA Account' > test/data/ica.out
+	-a 'Assets:ICA Account' -af "# kr" -ds "," -gs "." > test/data/ica.out
```

Comment:
You can't supply "# kr" here as an option, the point of garbage being that the user does not know (or care) about what it is before receiving the data. Conversion must work without requiring such precise a-priori specs given by the user.